src/game_inventory.cpp

```diff
@@ -1534,7 +1534,10 @@ class bionic_install_preset: public inventory_selector_preset
             } else if( pa.is_npc() && !bid->npc_usable ) {
                 return _( "CBM not compatible with patient" );
             } else if( !p.has_enough_anesth( itemtype ) ) {
-                return string_format( _( "%i mL" ), itemtype->bionic->difficulty * 40 );
+                const int weight = units::to_kilogram( g->u.bodyweight() ) / 10;
+                const requirement_data req_anesth = *requirement_id( "anesthetic" ) *
+                                                    loc.get_item()->type->bionic->difficulty * 2 * weight;
```

Comment:
This `* 2` appears all the time, it should probably be part of the requirement itself.

Reply:
Is it better now? It's `*2` instead of `*20` because requierment don't accept float so to have a rate of 0.03 I have to tweak the rest of the formula to keep using int